checks/check_status.py

```diff
@@ -290,7 +290,7 @@ def _get_pickle_path(cls):
             path = PidFile.get_dir()
         else:
             path = tempfile.gettempdir()
-        return os.path.join(path, cls.__name__ + '.pickle')
+        return os.path.join(path, prefix + cls.__name__ + '.pickle')
```

Comment:
filename becomes: `splunkCheckData.pickle`, shall we prefix `events` in there as well? Since CheckData is defined to be reusable it might be reused in other checks as well, let's make it explicit and avoid possible conflicts (if at all probable).